packages/atomic/src/components/result-template-components-v1/atomic-result-multi-text/atomic-result-multi-text.tsx

```diff
@@ -0,0 +1,164 @@
+import {Component, Prop, h, State, VNode} from '@stencil/core';
+import {
+  BreadcrumbManager,
+  buildBreadcrumbManager,
+  Result,
+  ResultTemplatesHelpers,
+} from '@coveo/headless';
+import {ResultContext} from '../../result-template-components/result-template-decorators';
+import {getFieldValueCaption} from '../../../utils/field-utils';
+import {
+  Bindings,
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {titleToKebab} from '../../../utils/utils';
+
+const listItemClasses = 'inline-block';
+
+/**
+ * The `atomic-result-multi-text` component renders the values of a multi-value string result field.
+ * @part result-multi-text-separator - The separator between two field values.
+ * @part result-multi-text-value - A field value.
+ * @slot result-multi-text-value-* - Allows the personalization of a particular field value based on its indexed name.
+ */
+@Component({
+  tag: 'atomic-result-multi-text-v1',
```

Comment:
There are no `atomic-result-multi-text` v0 component, so the `-v1` is not really needed here

Reply:
I think it might be helpful if we included the term "multi-value" in the name, since this is how it appears in other places like the AdminUI, the JSUI docs etc.

Maybe: `atomic-result-multi-value-text`